ecs-cli/modules/commands/local/local_command.go

```diff
@@ -164,33 +164,33 @@ func flagName(longName string) string {
 func flagDescription(longName, cmdName string) string {
 	m := map[string]map[string]string{
 		flags.TaskDefinitionCompose: {
-			upCmdName: "The Compose file `name` of a task definition to run.",
+			upCmdName: "Specifies the filename `value` that contains the Docker Compose content to run locally.",
 		},
 		flags.TaskDefinitionFile: {
-			createCmdName: fmt.Sprintf("The file `name` of a task definition json to convert. If not specified, defaults to %s", project.LocalInFileName),
-			upCmdName:     fmt.Sprintf("The file `name` of a task definition json to convert and run. If not specified, defaults to %s", project.LocalInFileName),
-			psCmdName:     "List all running containers matching the task definition file `name`.",
-			downCmdName:   "Stop and remove all running containers matching the task definition file `name`.",
+			createCmdName: fmt.Sprintf("Specifies the filename `value` that contains the task definition JSON to convert to a Docker Compose file. If one is not specified, the ECS CLI will look for %s", project.LocalInFileName),
+			upCmdName:     fmt.Sprintf("Specifies the filename `value` containing the task definition JSON to convert and run locally.  If one is not specified, the ECS CLI will look for %s", project.LocalInFileName),
+			psCmdName:     "Lists all running containers matching the task definition filename `value`.",
```

Comment:
nit: these two lines end with a '.' whereas the previous two don't :P can we end each description with a dot?

Reply:
Sure, happy to. I actually explicitly removed the dot from these sense they end with filenames, and the previous version omitted dots after filenames. But I can add them.